<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindCostumerPage</class>
 <widget class="QDialog" name="FindCostumerPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#FindCostumerPage{
background-color: #eaf2f8;
}</string>
  </property>
  <widget class="QListWidget" name="userNameList">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>200</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#userNameList {
	color: rgb(0, 0, 0);
        background-color: #f8f9fa;
        border: 1px solid #bdc3c7;
        font-size: 14px;
        padding: 4px;
    }
    #userNameList::item {
        padding: 8px;
        border-bottom: 1px solid #dcdcdc;
    }
    #userNameList::item:selected {
        background-color: #d6eaf8;
        color: #154360;
    }</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>580</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> #searchButton {
        background-color: #2980b9;
        color: white;
        border: none;
        border-radius: 6px;
        padding: 8px 16px;
        font-size: 14px;
    }
    #searchButton:hover {
        background-color: #3498db;
    }
    #searchButton:pressed {
        background-color: #1c5980;
    }</string>
   </property>
   <property name="text">
    <string>جستجو</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userNameField">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#userNameField {
        border: 1px solid #2c3e50;
        border-radius: 6px;
        padding: 6px;
        background-color: #ecf0f1;
        font-size: 14px;
	    color: rgb(0, 0, 0);
 }
 #userNameField:focus {
        border: 1px solid #2980b9;
        background-color: #ffffff;
    }</string>
   </property>
  </widget>
  <widget class="QListWidget" name="costumersList">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>200</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#costumersList {
	color: rgb(0, 0, 0);
        background-color: #f8f9fa;
        border: 1px solid #bdc3c7;
        font-size: 14px;
        padding: 4px;
    }
    #costumersList::item {
        padding: 8px;
        border-bottom: 1px solid #dcdcdc;
    }
    #costumersList::item:selected {
        background-color: #d6eaf8;
        color: #154360;
    }</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label {
        color: #2c3e50;
        font-size: 14px;
        font-weight: bold;
    }</string>
   </property>
   <property name="text">
    <string>نام کاربری مشتری مدنظر را وارد کنید</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
